### DISCUSSION TITLE
New Destination: Trino

### DISCUSSION THREAD

**Discussion by GrazianoFracasso** [airbytehq/airbyte#33161] [New Connector Request]
### Connector Name

Trino

### What type of integration

Destination

### Relevant Information

Can you please add a trino source/destination?

### Contribute

- [ ] Yes, I want to contribute

**Comment by remisharoon** [comment]
its surprising trino/presto is not in the list. its even more surprising nobody is inetrested?
why?

**Reply by marcosmarxm** [reply]
I moved the issue to a discussion last month only :)

**Comment by kempspo** [comment]
I wonder if it's possible to reuse the [Starburst destination](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/destination-starburst-galaxy) though it currently only supports S3 (and my use case is GCS)?

**Comment by krutoileshii** [comment]
would love to see this, and yes , reusing/extending that might be fastest way to get there
